src/sentry/incidents/logic.py

```diff
@@ -459,7 +460,16 @@ class AlertRuleNameAlreadyUsedError(Exception):
 
 # Default values for `SnubaQuery.resolution`, in minutes.
 DEFAULT_ALERT_RULE_RESOLUTION = 1
-DEFAULT_CMP_ALERT_RULE_RESOLUTION = 2
+DEFAULT_CMP_ALERT_RULE_RESOLUTION_MULTIPLIER = 2
```

Comment:
what does CMP stand for?

Reply:
comparison?

Reply:
Yes, fwiw, all i did was add `_MULTIPLIER`